### PR TITLE
Some issues related with logging

### PR DESCRIPTION
**Pull request by fgalan**
FIX error level traces ignoring -logLevel NONE
FIX wrong over-logging at error level updating attributes having metadata without type

@kzangeli 
@crbrox

### CODE REVIEW

File: src/lib/mongoBackend/MongoCommonUpdate.cpp [hunk 1]
```diff
@@ -321,23 +322,33 @@ static bool matchMetadata(BSONObj& md1, BSONObj& md2)
 static bool equalMetadataValues(BSONObj& md1, BSONObj& md2)
 {
 
+  bool md1TypeExist = md1.hasField(ENT_ATTRS_MD_TYPE);
+  bool md2TypeExist = md2.hasField(ENT_ATTRS_MD_TYPE);
 
-  // Same declared type ?
-  if (getField(md1, ENT_ATTRS_MD_TYPE).type() != getField(md2, ENT_ATTRS_MD_TYPE).type())
+  // If type exist in one metadata but not in the other, then result is unequality
```
Comment (kzangeli) [inline src/lib/mongoBackend/MongoCommonUpdate.cpp]: English:
exist => exists
then result => then the result

Comment (fgalan) [inline src/lib/mongoBackend/MongoCommonUpdate.cpp]: Fixed in 1bb10ae84d3640f74aac4698e097c12ebfe73609.

**Comment by kzangeli** [comment]
LGTM

**Comment by crbrox** [comment]
LGTM

**Pull request merged**